src/Database/Scope/Scope.php

```diff
@@ -93,7 +93,7 @@ public function appendToCacheKey($key)
      */
     public function applyToModel(Model $model)
     {
-        if (! $this->onlyScopeRelations && ! is_null($this->scope)) {
+        if (! $this->onlyScopeRelations) {
             $model->scope = $this->scope;
         }
```

Comment:
Why did you have to remove this?

It might be unnecessary. Just curious whether you had to remove it to make this work.

Reply:
No, keeping still works as expected. I added it back in.